Makefile

```diff
@@ -39,14 +40,41 @@ DFSET=$(PGO_BASEOS)
 # repository using docker (otherwise the images may not be recognized)
 export BUILDAH_FORMAT ?= docker
 
+DOCKERBASEREGISTRY=registry.access.redhat.com/
+
 # Allows simplification of IMGBUILDER switching
 ifeq ("$(IMGBUILDER)","docker")
-	IMGCMDSTEM=docker build
+        IMGCMDSTEM=docker build
+endif
+
+# Allows consolidation of ubi/rhel/centos Dockerfile sets
+ifeq ("$(PGO_BASEOS)", "rhel7")
+        DFSET=rhel
 endif
 
-# Allows consolidation of ubi/rhel Dockerfile sets
 ifeq ("$(PGO_BASEOS)", "ubi7")
-	DFSET=rhel7
+        DFSET=rhel
+endif
+
+ifeq ("$(PGO_BASEOS)", "rhel8")
+        DFSET=rhel
+        PACKAGER=dnf
+endif
```

Comment:
Per comment in CrunchyData/crunchy-containers#1261 I would remove this line.

Reply:
Will PR this change shortly